pyomo/solvers/plugins/solvers/GUROBI_RUN.py

```diff
@@ -11,8 +11,17 @@
 
 import re
 
+
+"""
+This script is run using the Gurobi/system python. Do not assume any third party packages
+are available!
+"""
 from gurobipy import *
-from six.moves import zip
+import sys
+if sys.version_info[0] < 3:
+    from itertools import izip
+    zip = izip
```

Comment:
I could nit-pick:
```
    from itertools import izip as zip
```

Reply:
You can make a simple cosmetic change like that by making a commit directly to the PR branch via the Github web interface.